_includes/alert-gdpr-disclaimer.md

```diff
@@ -0,0 +1,5 @@
+{% include alert.html type="warning" icon="warning" body="GDPR sensitive data
```

Comment:
Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6189093)

Reply:
@asbjornu What does this mean? :)

Reply:
It's a bug in Codacy. Just ignore.